### ISSUE TITLE
Warning: there are unknown plugin options for "gatsby-source-contentstack": downloadImages

### ISSUE THREAD

**Issue by josephmarkus** [contentstack/gatsby-source-contentstack#99]
Whenever I run `yarn start` I get:

```
warn Warning: there are unknown plugin options for "gatsby-source-contentstack": downloadImages
Please open an issue at ghub.io/gatsby-source-contentstack if you believe this option is valid.
```

When I look at the README of this repo, it appears a valid option 🤔

**Comment by josephmarkus** [comment]
My understanding is that `downloadImages` should be included in this validation check:

https://github.com/contentstack/gatsby-source-contentstack/blob/develop/src/gatsby-node.js#L420-L430

**Comment by vkalta** [comment]
@josephmarkus Sorry for the inconvenience caused. Will check this and update you asap. :)

**Comment by vkalta** [comment]
@josephmarkus This issue has been fixed on version 3.0.0.

Thanks.
